diff -up timezone-2026a/backward.extra-tz-links timezone-2026a/backward
--- timezone-2026a/backward.extra-tz-links	2024-07-03 12:17:49.000000000 +0200
+++ timezone-2026a/backward	2026-03-02 22:14:29.325471866 +0100
@@ -329,3 +329,18 @@ Link	Pacific/Honolulu	HST
 Link	America/Los_Angeles	PST8PDT
 Link	Pacific/Guadalcanal	Pacific/Ponape	#= Pacific/Pohnpei
 Link	Pacific/Port_Moresby	Pacific/Truk	#= Pacific/Chuuk
+# links requested by users (to access timezones by the name of main cities)
+# Canada/Saskatchewan 
+Link	America/Regina		America/Saskatoon
+# Canada/Mountain (Alberta)
+Link	America/Edmonton	America/Calgary
+# Canada/Atlantic (New Brunswick)
+Link	America/Halifax		America/Fredericton
+# PRC
+Link	Asia/Shanghai		Asia/Beijing
+# South Africa
+Link	Africa/Johannesburg	Africa/Pretoria
+# Switzerland
+Link	Europe/Zurich		Europe/Berne
+# And a little easter egg...
+Link	Europe/Zurich		Europe/Unterschaechen
